src/shared_modules/indexer_connector/src/indexerConnector.cpp

```diff
@@ -200,14 +200,12 @@ IndexerConnector::IndexerConnector(
                 }
                 catch (const std::exception& e)
                 {
-                    // Improved logging message
                     logWarn(IC_NAME,
-                            "Error initializing IndexerConnector for index '%s': %s. Retrying in %ld seconds. Maximum "
-                            "wait time: %ld seconds.",
+                            "Unable to initializing IndexerConnector for index '%s': %s. Retrying in %ld "
```

Comment:
This message should not be perpetual (printed every 30 seconds non-stop).

At the Warning level, we should only print it once informing that we are going to retry continuously until successful (which has to print a non-technical log ex: Initialized with success.)

The perpetual message can be at the DEBUG level, in terms of analyzing the case.